lib/src/main/java/org/kiwix/libkiwix/JNIKiwix.java_

```diff
@@ -18,11 +18,11 @@
  * MA 02110-1301, USA.
  */
 
-package org.kiwix.kiwixlib;
+package org.kiwix.libkiwix;
 
 import android.content.Context;
 import com.getkeepsafe.relinker.ReLinker;
-import org.kiwix.kiwixlib.JNIICU;
+import org.kiwix.libkiwix.JNIICU;
 
 public class JNIKiwix
```

Comment:
why we renaming the JNIKiwix.java to JNIKiwix.java_ (Because it's make it as file instead of java class. we can not access this class methods).

Reply:
Arf, I missed that. Good catch
The thing is that gradle/javac complain because it cannot find `android.content.Context` and `com.getkeepsafe.relinker.ReLinker`.
I've rename it temporally (and forget about it) to still be able to build the wrapper.

If you know how to add the two dependencies in gradle, you are more than welcomed.

Reply:
I've rename back the file to `JNIKiwix.java` and it compiles. I don't know why it was failing (but I had a lot of other error at this time)